services/ui-src/src/utils/other/formTemplate.test.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  mockFlattenedReportRoutes,
+  mockReportJson,
+} from "utils/testing/setupJest";
+import {
+  compileValidationJsonFromRoutes,
+  copyAdminDisabledStatusToForms,
+} from "./formTemplate";
+
+describe("Test copyAdminDisabledStatusToForms", () => {
```

Comment:
I ended up moving this code to the backend as part of that refactor—will move these tests into my PR once this is merged

Reply:
Oh, perfect!